src/NugetPackagesCommon.props

```diff
@@ -15,7 +15,9 @@
     <DefineConstants>TRACE;JETBRAINS_ANNOTATIONS</DefineConstants>
     <DebugType>full</DebugType>
     <DebugSymbols>True</DebugSymbols>
-    <LangVersion>7.3</LangVersion>
+    <LangVersion>8.0</LangVersion>
+    <Nullable>enable</Nullable>
+    <NullableContextOptions>enable</NullableContextOptions>

```

Comment:
dit is geen dubbelopisme?

Reply:
NullableContextOptions is omdat AppVeyor nog een oude compiler heeft.